bricks/aem-logo-wall/aem-logo-wall.js

```diff
@@ -0,0 +1,80 @@
+import { Brick } from '../../scripts/aem.js';
+
+function createTag(tag, attributes, html) {
+  const el = document.createElement(tag);
+  if (html) {
+    if (html instanceof HTMLElement) {
+      el.append(html);
+    } else {
+      el.insertAdjacentHTML('beforeend', html);
+    }
+  }
+  if (attributes) {
+    Object.keys(attributes).forEach((key) => {
+      el.setAttribute(key, attributes[key]);
+    });
+  }
+  return el;
+}
+
+export default class LogoWall extends Brick {
+  connectedCallback() {
+    const items = [...this.root.children];
+    const slot = this.shadowRoot.querySelector('slot');
```

Comment:
IIUC this code it's just rearranging its descendant elements to get the desired layout. Does that warrant using Shadow DOM? Not a blocker for merging but something to discuss generally.

Reply:
Shadow DOM is not needed, just used it as it's the "default" (what we're doing in the preexisting bricks), but it can be changed if need be.

Reply:
Ok, good, we'll need to discuss that when we review this first pass. Maybe we need two base classes, one for shadow and one for plain HTML components, or no base class for the latter.